nfadeeva/assignment1/Assignment_1_Q2.py

```diff
@@ -0,0 +1,45 @@
+class Node:
+    def __init__(self, data):
+        self.data = data
+        self.next = None
+
+
+class LinkedList:
+    def __init__(self):
+        self.head = None
+
+    def push(self, value):
+        """
+        Push value to the head of the list
+        """
+
+        new_node = Node(value)
+        new_node.next = self.head
+        self.head = new_node
+
+    def find_kth_last_element(self, k):
```

Comment:
nit: as it's a method of the LinkedList I would just call it "kth_last_element(...)".